generated-platform-project/quarkus-optaplanner/bom/pom.xml

```diff
@@ -81,57 +81,63 @@
       <dependency>
         <groupId>org.drools</groupId>
         <artifactId>drools-alphanetwork-compiler</artifactId>
-        <version>7.59.0.Final</version>
+        <version>8.13.0.Beta</version>
+      </dependency>
+      <dependency>
+        <groupId>org.drools</groupId>
+        <artifactId>drools-bom</artifactId>
+        <version>8.13.0.Beta</version>
+        <type>pom</type>
```

Comment:
Are the Beta versions expected? Will it stay for the final platform release?

Reply:
yes it is expected. We are transitioning the Drools project in the Kogito pipelines and will have a few release of Drools as Beta release

Reply:
Thanks for the info.